src/serac/physics/tests/solid_nonlinear_solve.cpp

```diff
@@ -56,59 +277,52 @@ void functional_solid_test_nonlinear_buckle()
-                                                  .max_line_search_iterations = 20,
-                                                  .print_level                = 1};
-
-  serac::LinearSolverOptions linear_options = {.linear_solver  = LinearSolver::CG,
-                                               .preconditioner = Preconditioner::HypreAMG,
-                                               .relative_tol   = 1.0e-6,
-                                               .absolute_tol   = 1.0e-12,
-                                               .max_iterations = 3 * Nx * Ny * Nz};
+  auto [nonlinear_options, linear_options] = get_opts(3 * Nx * Ny * Nz, 1e-11);
 
   auto seracSolid = std::make_unique<seracSolidType>(
       nonlinear_options, linear_options, serac::solid_mechanics::default_quasistatic_options,
       serac::GeometricNonlinearities::On, "serac_solid", meshTag, std::vector<std::string>{});
 
-  // set linear elastic material
   serac::solid_mechanics::NeoHookean material{density, bulkMod, shearMod};
-  // serac::solid_mechanics::StVenantKirchhoff material{density, bulkMod, shearMod};
   seracSolid->setMaterial(serac::DependsOn<>{}, material);
 
   // fix displacement on side surface
   seracSolid->setDisplacementBCs({2, 3, 4, 5}, [](const mfem::Vector&, mfem::Vector& u) { u = 0.0; });
+  // seracSolid->setDisplacementBCs({3}, [](const mfem::Vector&, mfem::Vector& u) { u = 0.0; });
 
   serac::Domain topSurface = serac::Domain::ofBoundaryElements(*meshPtr, serac::by_attr<DIM>(6));
-  seracSolid->setTraction([&](auto, auto n, auto) { return -loadMagnitude * n; }, topSurface);
-
+  // seracSolid->setTraction([&](auto, auto n, auto) { return -loadMagnitude * n; }, topSurface);
+  seracSolid->setPressure([&](auto, auto) { return loadMagnitude; }, topSurface);
   seracSolid->completeSetup();
   seracSolid->advanceTimestep(1.0);
 
-  const serac::FiniteElementState& displacement = seracSolid->state("displacement");
-  mfem::ParGridFunction            uGF(const_cast<mfem::ParFiniteElementSpace*>(&displacement.space()));
-  displacement.fillGridFunction(uGF);
-
-  mfem::VisItDataCollection visit_dc("nonlinearPlate", meshPtr);
-  visit_dc.RegisterField("u", &uGF);
-  visit_dc.Save();
+  seracSolid->outputStateToDisk("paraview_buckle_easy");
 }
 
-TEST(SolidMechanics, nonlinear_solve) { functional_solid_test_nonlinear_buckle(); }
+TEST(SolidMechanics, nonlinear_solve_buckle_easy) { functional_solid_test_nonlinear_buckle(5e-10); }
+// TEST(SolidMechanics, nonlinear_solve_buckle_medium) { functional_solid_test_nonlinear_buckle(4e-4); }
+// TEST(SolidMechanics, nonlinear_solve_buckle_hard) { functional_solid_test_nonlinear_buckle(3e-2); }
+// TEST(SolidMechanics, nonlinear_solve_euler) { functional_solid_test_euler(); }
 
 int main(int argc, char* argv[])
 {
+  axom::CLI::App app{"Nonlinear problems"};
+  app.add_option("-p", mesh_path, "Mesh file to use")->check(axom::CLI::ExistingDirectory);
```

Comment:
This option and variable name are not clear that it should point to a file or a directory. It is also not used anywhere unless I am missing something which is definitely possible. Can this be clarified or just removed entirely?

Reply:
commented, as I do use it a lot with my local branch that has more performance tests